### ISSUE TITLE
Option to "lock" brightness slider with night light slider

### ISSUE THREAD

**Issue by orschiro** [TimurKiyivinski/gnome-shell-night-light-slider-extension#25]
Dear @TimurKiyivinski,

I tend to align the night light slider with the brightness one. 

Keeping both at the same level provides a very good combination, I found!

Looking forward to hearing what you think from your experiences. 

Hence, I would welcome an option in the settings to lock brightness and night light slider. 

Locking would make both move simultaneously when moving the night light slider. 

What do you think?

**Comment by kiyui** [comment]
I'm curious what behaviour you're after with this. I understand that the night light slider should lock with the brightness slider, but I want your opinion on these options and if they were implied:

> Should updating the brightness slider affect the night light?

It could be the case that I may want to adjust the night light and brightness together, then slightly alter brightness without altering night light. What should happen in this scenario?

> Should the night light on start default to the value of the brightness?

To have them in sync on shell startup, I may need to have the night light value be set to that of brightness or vice-versa, or just leave it as is, unsynced at first. What do you think?

Basically, if you want to sync night light intensity with brightness, should it be possible for the beightness to differ from the night light value? Would love to hear your input, thanks :smile:

**Comment by orschiro** [comment]
Hey @TimurKiyivinski, 

Thanks for your interesting questions! Before I dive into them, do you find yourself in the same situation that the both bars are often at more or less the same position depending on the time of the day?

Give you an example. Now it's 09:00 AM and my two sliders are almost in sync to produce a vision that I find to work well for me:

![image](https://user-images.githubusercontent.com/170947/39857286-16670c64-5433-11e8-9993-b30d5cd5c877.png)

**Comment by kiyui** [comment]
Hey @orschiro, I don't have this as my brightness usually depends on the amount of ambient light in the room. My "night light" however I have on the whole day unless I need to make the screen less red-shifted for presentations/pair-programming at work, etc. so this does not apply to me.

Testing them to be the same however does produce interesting results that I never noticed before and does seem very natural.

**Comment by orschiro** [comment]
> Testing them to be the same however does produce interesting results that I never noticed before and does seem very natural.

Yes, fully agree! This is what I found, too. Was surprised at first and that's why I opened this issue! :-D

**Comment by kiyui** [comment]
I've implemented the changes and [published a release](https://github.com/TimurKiyivinski/gnome-shell-night-light-slider-extension/releases/tag/9.0) but am leaving this open if there are any additional suggestions as to how to handle this functionality.

Perhaps you want to use the tagged version instead if you want to test the new functionality, as it seems that the previous release has not even been reviewed. Not sure what gives, since that was ages ago.

**Comment by orschiro** [comment]
@TimurKiyivinski wow just tested the new brightness sync and super impressed by the implementation. Love scrolling over the night light icon to adjust both at the same time! :-D

Thanks so much!

**Comment by orschiro** [comment]
Btw the extension review system is strange. Just received an info to upgrade albeit I am using the latest release from Github which would essentially downgrade me...

![image](https://user-images.githubusercontent.com/170947/40046920-5ddf034a-582e-11e8-8fb7-5128ca054b1e.png)

**Comment by kiyui** [comment]
Yeah that would be because I don't tag the `version` inside the metadata when I publish to GitHub so the GNOME extension page thinks it's outdated. The reason for which being that when I upload it to the extensions page on GNOME, they automatically bump the `version` there which I've had issues with in the past.

Perhaps you can modify the `metadata.json` locally and add a `version` key as `8.0` :grin:
Perhaps I should upload releases with updated `version` tags :thinking: 

Also, closing this as fixed since you like it~

**Comment by orschiro** [comment]
> Perhaps you can modify the metadata.json locally and add a version key as 8.0 😁
Perhaps I should upload releases with updated version tags 🤔

Ok good idea, thanks!
